Cordova.podspec

```diff
@@ -0,0 +1,32 @@
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
+  s.version      = "5.1.0"
+  s.summary      = "Cordova for iOS"
```

Comment:
I think this should say "Apache Cordova for iOS"

Reply:
Will do.